src/cluster.c

```diff
@@ -873,9 +873,19 @@ void clusterCommand(client *c) {
     } else if (!strcasecmp(c->argv[1]->ptr,"slots") && c->argc == 2) {
         /* CLUSTER SLOTS */
         clusterCommandSlots(c);
-    } else if (!strcasecmp(c->argv[1]->ptr,"shards") && c->argc == 2) {
-        /* CLUSTER SHARDS */
-        clusterCommandShards(c);
+    } else if (!strcasecmp(c->argv[1]->ptr,"shards") &&
+                (c->argc == 2 || c->argc == 3))
+    {
+        /* CLUSTER SHARDS [TOPOLOGY] */
+        int topology = 1;
```

Comment:
Usually, I set this kind of bool variable default value as 0 (here the variable more close to a bool variable).
But it is not a big issue I think